Modules/@babylonjs/react-native/android/src/main/cpp/BabylonNativeInterop.cpp

```diff
@@ -79,10 +79,10 @@ extern "C" JNIEXPORT void JNICALL Java_com_babylonreactnative_BabylonNativeInter
 
 extern "C" JNIEXPORT void JNICALL Java_com_babylonreactnative_BabylonNativeInterop_00024BabylonNative_setPointerButtonState(JNIEnv* env, jclass obj, jint pointerId, jint buttonId, jboolean isDown, jint x, jint y)
```

Comment:
`buttonId` is no longer used, will you remove this?

Reply:
yes